plugins/apache_http_logs.yaml

```
@@ -0,0 +1,182 @@
+version: 0.0.1
+title: Apache HTTP Server
+description: Log parser for Apache HTTP Server
+parameters:
+  - name: log_format
+    description: When choosing the 'default' option, the agent will expect and parse logs in a format that matches the default logging configuration. When choosing the 'observIQ' option, the agent will expect and parse logs in an optimized JSON format that adheres to the observIQ specification, requiring an update to the apache2.conf file. See the Apache HTTP Server source page for more information.
```

I think we need to embed the info about the observIQ format here, in a comment or something, like how it was done in the nginx plugin. The place where this is documented is so obscure.

---

```
 See the Apache HTTP Server source page for more information.
```
I don't think this makes sense to anyone without context of other things we make. We should remove this part.
```suggestion
  - name: log_format
    description: When choosing the 'default' option, the agent will expect and parse logs in a format that matches the default logging configuration. When choosing the 'observIQ' option, the agent will expect and parse logs in an optimized JSON format that adheres to the observIQ specification, requiring an update to the apache2.conf file.
```